Remotion/Core/Reflection.CodeGeneration.TypePipe/IPipelineSettingsProvider.cs

```diff
@@ -1,25 +1,29 @@
 ﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
 // Copyright (c) rubicon IT GmbH, www.rubicon.eu
-// 
-// The re-motion Core Framework is free software; you can redistribute it 
-// and/or modify it under the terms of the GNU Lesser General Public License 
-// as published by the Free Software Foundation; either version 2.1 of the 
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
 // License, or (at your option) any later version.
-// 
-// re-motion is distributed in the hope that it will be useful, 
-// but WITHOUT ANY WARRANTY; without even the implied warranty of 
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 // GNU Lesser General Public License for more details.
-// 
+//
 // You should have received a copy of the GNU Lesser General Public License
 // along with re-motion; if not, see http://www.gnu.org/licenses.
-// 
+//
 using System;
-using System.Configuration;
+using Remotion.TypePipe;
 
-namespace Remotion.Reflection.CodeGeneration.TypePipe.Configuration
+namespace Remotion.Reflection.CodeGeneration.TypePipe
 {
-  public class EnableSerializationWithoutAssemblySavingConfigurationElement : ConfigurationElement
+  /// <summary>
+  /// Defines the API required for retrieving <see cref="PipelineSettings"/>.
+  /// </summary>
+  public interface IPipelineSettingsProvider
```

Comment:
I prefer to have a seealso-element in the xmldoc listing the implementations. Helps with doc-navigation.